src/name_bazaar/ui/pages/user_offerings_page.cljs

```diff
@@ -53,7 +54,9 @@
 
 (defn user-offerings []
   (let [search-results (subscribe [:offerings/user-offerings])
-        route-params (subscribe [:district0x/route-params])]
+        route-params (subscribe [:district0x/route-params])
+        my-address (subscribe [:my-full-resolved-address])
```

Comment:
Please don't use term `my-address` as it's bit misleading. It's only `active-address` or `my-addresses`